reference/fleet/secure-connections.md

```diff
@@ -268,6 +270,38 @@ To encrypt traffic between {{agent}}s, {{fleet-server}}, and {{es}}:
         `certificate-authorities`
         :   CA certificate to use to connect to {{fleet-server}}. This is the CA used to [generate a certificate and key](#generate-fleet-server-certs) for {{fleet-server}}.
 
-        Don’t have an enrollment token? On the **Agents** tab in {{fleet}}, click **Add agent**. Under **Enroll and start the Elastic Agent**, follow the in-product installation steps, making sure that you add the `--certificate-authorities` option before you run the command.
+        Don’t have an enrollment token? On the **Agents** tab in {{fleet}}, select **Add agent**. Under **Enroll and start the Elastic Agent**, follow the in-product installation steps, making sure that you add the `--certificate-authorities` option before you run the command.
+
+
+## Configure SSL/TLS using {{kib}} [fleet-server-ssl-ui-settings]
+```{applies_to}
+  stack: ga 9.1
+```
+
+You can configure SSL/TLS settings for {{fleet-server}} hosts directly in the {{fleet}} UI, without relying on CLI flags or policy overrides.
+
+To access these settings:
+
+1. In **Kibana**, go to **Management > {{fleet}} > Settings**.
+2. Under **{{fleet-server}} hosts**, select **Add host** or edit an existing host.
+3. Expand the **SSL options** section.
+
+### SSL options
+
+These are the available UI fields and their CLI equivalents:
+
+| **UI Field**                                      | **CLI Flag**                          | **Purpose** |
+|--------------------------------------------------|---------------------------------------|-------------|
+| Client SSL Certificate                           | `--elastic-agent-cert`               | {{agent}} client certificate to use with {{fleet-server}} during mTLS authentication. |
+| Client SSL Certificate key                       | `--elastic-agent-cert-key`           | {{agent}} client private key to use with {{fleet-server}} during mTLS authentication. This field uses secret storage and requires {{fleet-server}} v8.12.0 or later. You can optionally choose to store the value as plain text instead. |
```

Comment:
I think that the these should be `-fleet-server-es-cert` and `-fleet-server-es-cert-key`. The `--elastic-agent-cert` are the ones available under the `output` section of fleet settings